src/types/chunks/ITransaction.ts

```diff
@@ -36,3 +36,11 @@ export type ITransactionLog = {
   /** Data of the log */
   data: string;
 };
+
+export interface IParsedTransaction {
```

Comment:
ethers already has a type for it, the return type in the parseTransaction method on the ProtocolHelper returns that